arch/eventloop_lws_http.c

```diff
@@ -0,0 +1,259 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*
+*   Copyright 2023 (c) Fraunhofer IOSB (Author: Noel Graf)
+*/
+
+/* The HTTP ConnectionManager is architecture agnostic. It selects an existing
+* TCP ConnectionManager and uses it for the underlying TCP connections. */
+
+/* Suppress the warning 'redefinition of typedef' in the libwebsockets library.
+ * Removed as soon as the patch is included in the main. */
+#pragma GCC diagnostic ignored "-Wpedantic"
```

Comment:
Which patch included in the main?
The main of libwebsockets?
Refer to the patch so the status can be tracked.